src/Utilities.js

```diff
@@ -76,3 +76,27 @@ function extend_(destination, source) {
   }
   return destination;
 }
+
+/* exported getValueCaseInsensitive_ */
+/**
+ * Gets the value stored in the object under the given key, in a
```

Comment:
`getValueCaseInsensitive_` seems like an obscure method.

Why don't we make a util method that converts an object to a lowercased-keyed object and then get that value?